### ISSUE TITLE
winstall shows removed packages

### ISSUE THREAD

**Issue by infinitepower18** [MehediH/winstall#125]
Noticed [Windscribe](https://winstall.app/apps/WindscribeLimited.Windscribe), which does not exist in the repository anymore but continues to show on this site. Most likely this issue occurs for all removed packages.

**Comment by KnifMelti** [comment]
FileZilla...

**Comment by masterflitzer** [comment]
FileZilla got removed, Windscribe was renamed from `WindscribeLimited.Windscribe` to `Windscribe.Windscribe` and .NET was renamed from `Microsoft.dotnet` to `Microsoft.DotNet.6` (and many more different versions)

I think winstall is caching everything and just adding new ones without removing old ones

**Comment by DanielHabenicht** [comment]
yep same happens with https://winstall.app/apps/YairAichenbaum.Files which is non existent anymore

**Comment by st-jliu** [comment]
Alread fix that, incremental update will auto remove no version associated apps
